nixos/modules/programs/direnv.nix

```diff
@@ -0,0 +1,136 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}: let
+  cfg = config.programs.direnv;
+in {
+
+  options.programs.direnv = {
+
+    enable = lib.mkOption {
```

Comment:
The is a util function `mkEnableOption` for bool option.

Example: https://github.com/NixOS/nixpkgs/blob/873012e0ac6c4feb66eeb88d7e14df4a8323f565/nixos/modules/services/hardware/kanata.nix#L142

Reply:
Added for enable